app/api/utils.py

```diff
@@ -99,3 +136,48 @@ def validate_office(request):
         errors.append(error)
 
     return errors
+
+
+"""Decorators"""
+
+
+def login_required(fn):
+    """Decorator to protect admin route"""
+    @wraps(fn)
+    def token_decorator(*args, **kwargs):
+
+        if "Authorization" in request.headers:
+            token = request.headers['Authorization']
+        else:
+            return make_response(jsonify({"error": "Token is missingrequired for this method"}), 403)
+
+        try:
+            data = jwt.decode(token, os.getenv('SECRET_KEY'))
+        except Exception as e:
+            return make_response(jsonify({"message": "Token Required"}))
+
+        return fn(*args, **kwargs)
+    return token_decorator
+
+
+def admin_required(fn):
+    """Decorator to protect admin routes"""
+    @wraps(fn)
+    def admin_decorator(*args, **kwargs):
+
+        if "Authorization" in request.headers:
+            token = request.headers['Authorization']
+        else:
+            return make_response(jsonify({"error": "Admin Token is required for this method missing"}), 403)
+
+        try:
+            data = jwt.decode(token, os.getenv('SECRET_KEY'))
+            if data['isAdmin'] != True:
```

Comment:
First, verify what is your data by printing out data. `print(data)`. From there you will be able to parse the keys correctly.